### PR TITLE
Align PSU CLI output with SONiC CLI output template

### PR DESCRIPTION
**Pull request by andrii-savka**
Change the format of PSU CLI to align it with SONiC CLI output template

### CODE REVIEW

**Comment by andrii-savka** [comment]
The CLI output:

Previous output command:
+-------+----------+
| PSU     | Status     |
+====+======+
| PSU 1  | OK          |
+-------+----------+
| PSU 2 | NOT OK   |
+-------+----------+

Actual updated output:
(aligned to other SONiC CLI utilities)
_PSU    Status
-----  --------
PSU 1  OK
PSU 2  NOT OK

File: psuutil/main.py [hunk 1]
```diff
@@ -172,7 +172,8 @@ def status(index, textonly):
         msg = ""
         psu_name = "PSU {}".format(psu)
         if psu not in supported_psu:
-            status_table.append([psu_name, "NOT SUPPORTED"])
+            print ("Error! The {} is not available on the platform.\n" \
```
Comment (jleveque) [inline psuutil/main.py]: Use `click.echo` instead of `print`

File: psuutil/main.py [hunk 2]
```diff
@@ -182,11 +183,14 @@ def status(index, textonly):
...
 
+    if not status_table:
+        return
+
     if textonly:
         for status in status_table:
             print status[0] + ':' + status[1]
     else:
-        print tabulate(status_table, header, tablefmt="grid")
+        print tabulate(status_table, header, tablefmt="simple")
```
Comment (jleveque) [inline psuutil/main.py]: Use `click.echo` instead of `print`

Also please do the same on lines 135 and 153.

File: psuutil/main.py [hunk 3]
```diff
@@ -182,11 +183,14 @@ def status(index, textonly):
             msg = 'NOT PRESENT'
         status_table.append([psu_name, msg])
 
+    if not status_table:
+        return
+
     if textonly:
```
Comment (jleveque) [inline psuutil/main.py]: Now that table format is simplified, do we need a "textonly mode"?

File: psuutil/main.py (hunk 1 above)
Comment (andrii-savka) [reply psuutil/main.py]: Replaced all "print" to "click.echo"

File: psuutil/main.py (hunk 3 above)
Comment (andrii-savka) [reply psuutil/main.py]: No, it doesn't needed because of SNMP uses PSU plugin directly. 
Removed "textonly mode

File: psuutil/main.py (hunk 2 above)
Comment (andrii-savka) [reply psuutil/main.py]: Replaced all "print" to "click.echo"

File: psuutil/main.py [hunk 4]
```diff
@@ -172,7 +171,8 @@ def status(index, textonly):
         msg = ""
         psu_name = "PSU {}".format(psu)
         if psu not in supported_psu:
-            status_table.append([psu_name, "NOT SUPPORTED"])
+            click.echo("Error! The {} is not available on the platform.\n" \
+            "Number of supported PSU - {}.".format(psu_name, platform_psuutil.get_num_psus()))
```
Comment (jleveque) [inline psuutil/main.py]: Change to `"Number of supported PSU: {}"`

Replacing hyphen and preceding space with a colon and removing the trailing period will make parsing the output easier, if need be.

Comment (jleveque) [reply psuutil/main.py]: Too late; already merged. Can modify later.

Comment (andrii-savka) [reply psuutil/main.py]: Thanks.
It does not needed now.

**Comment by andrii-savka** [comment]
The changes contains an issue which was fixes in following PR:
https://github.com/Azure/sonic-utilities/pull/164

**Pull request merged**